components/support/migration/metrics.yaml

```diff
@@ -0,0 +1,508 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$schema: moz://mozilla.org/schemas/glean/metrics/1-0-0
+
+migration:
+  migration_versions:
+    type: labeled_string
+    description: >
+      Versions of the migrations which were executed.
+    labels:
+      - history
+      - bookmarks
+      - logins
+      - open_tabs
+      - fxa
+      - gecko
+      - settings
+      - telemetry_identifiers
+    send_in_pings:
+      - migration
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/5888
+    data_reviews:
+      - https://github.com/mozilla-mobile/android-components/pull/5483
```

Comment:
You should add a link to the data review response here. Do not forget to update this and all the other links in this file.